app/views/users/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>A user homepage</h1>
\ No newline at end of file
+<div class='show-container'>
+    <div class='single-user'>
+        <div class='user-info'><%= render @user %></div>
+        <div class='single-user-info'>
+            <h3>Bio</h3>
+            <p><%= @user.bio %></p>
+        </div>
+        <h3 class='recent'>Recent Posts</h3>
+        <% @user.recent_posts.each_with_index do |post, index| %>
+            <%= render post, index: index %>
+        <% end %>
+    </div>
+    <button type='button' class='all-post-btn'>See all posts</button>
```

Comment:
- [ ] [REQUIRED] The button or link should redirect to all posts path. See the suggestions below

`<%= link_to 'See all Post', user_posts_path(@user) %>`